### PR TITLE
fix: fix failure behavior

### PR DESCRIPTION
**Pull request by maelle**
@Bisaloo @zkamvar 

I recently realized this part of the code was wrong (thanks to a failure!). If you use gitcellar, do you have an opinion on this?

### CODE REVIEW

**Comment by maelle** [comment]
@zkamvar would you have time to review this? I'd like to merge it soon, I use the fork in rOpenSci workflows as otherwise failures realllly fail. :sweat_smile:

**Comment by maelle** [comment]
I'll still welcome feedback later. In any case this can only be better than the former version so please update :-) (A problem might be my assumption the whole thing doesn't take more than 90 minutes, which is at least true for rOpenSci)

File: R/organization.R [hunk 1]
```diff
@@ -59,16 +59,19 @@ download_organization_repos <- function(organizations = NULL,
 
   message(sprintf("%s archives to be saved.", length(repos)))
 
-  while (length(repos) > 0 && time_spent(start_time = start_time) < 120) {
+  while (length(repos) > 0 && time_spent(start_time = start_time) < 90) {
+    Sys.sleep(600)
     status <- purrr::map_chr(repos, function(repo) repo$migration_state)
     ready_repos <- repos[status == "exported"]
     purrr::walk(ready_repos, function(repo) repo$launch_download())
     repos <- repos[status != "exported"]
```
Comment (Bisaloo) [inline R/organization.R:67]: One possible minor issue I see here is if a repo status changes between L64 and L66 (is this possible?). It seems slightly more robust to check which repo has `"exported"` status once at the start of the loop and re-use this variable.

**Comment by zkamvar** [comment]
I'm sorry for dragging this on. I trust your judgement and I'll test it out for my use-case next week 🤞🏼

File: R/organization.R (hunk 1 above)
Comment (Bisaloo) [reply R/organization.R:67]: Ah no, nevermind, you're already doing this since status is computed just once. Then compared twice but it doesn't matter.

**Pull request merged**